### PR TITLE
fix release tasks

### PR DESCRIPTION
**Pull request by ihar-kahadouski**

### CODE REVIEW

File: build-docker.gradle [hunk 1]
```diff
@@ -25,29 +25,84 @@ buildscript {
...
-import org.apache.tools.ant.filters.ReplaceTokens
+import com.bmuschko.gradle.docker.tasks.image.Dockerfile
 
 docker {
-    url.set("${dockerServerUrl}")
+    url.set(dockerServerUrl)
+}
+
+def isRelease = project.hasProperty('registryRelease')
+def devPostfix = isRelease ? '' : '-dev-5'
```
Comment (pbortnik) [inline build-docker.gradle]: lets name the variable as 'imagePostfix' and leave '-dev'

File: build-docker.gradle [hunk 2]
```diff
@@ -25,29 +25,84 @@ buildscript {
...
+           'description': "$project.description",
+           'maintainer' : "$maintainer"]
+    )
+    if (project.hasProperty('dockerPrepareEnvironment') && dockerPrepareEnvironment?.trim()) {
+        runCommand(dockerPrepareEnvironment)
+    }
+    environmentVariable(
+            ['JAVA_OPTS'       : project.hasProperty('dockerJavaOpts') ? dockerJavaOpts : '',
+             'ARTIFACT'        : artifact,
+             'APP_DOWNLOAD_URL': "https://dl.bintray.com/ihar-kahadouski/reportportal/com/epam/reportportal/${project.name}/${project.version}/${artifact}"]
```
Comment (pbortnik) [inline build-docker.gradle]: url contains 'ihar-kahadouski'

File: build-docker.gradle [hunk 3]
```diff
@@ -25,29 +25,84 @@ buildscript {
...
+    )
+    runCommand('sh -c "echo $\'#!/bin/sh \\n\\\n' +
+            'exec java $JAVA_OPTS -jar $ARTIFACT\' > /start.sh && chmod +x /start.sh"')
+    volume('/tmp')
+    runCommand('wget $APP_DOWNLOAD_URL')
+    exposePort(8080)
+    entryPoint('/start.sh')
```
Comment (pbortnik) [inline build-docker.gradle]: can you provide here in response a dockerfile that is generated from this?
i can't see installing of certs and fonts

Comment (ihar-kahadouski) [reply build-docker.gradle]: in gradle props

File: build-docker.gradle (hunk 2 above)
Comment (ihar-kahadouski) [reply build-docker.gradle]: fixed

File: build-docker.gradle (hunk 1 above)
Comment (ihar-kahadouski) [reply build-docker.gradle]: done

**Pull request merged**